package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "gameframe",
+  "version": "1.0.4",
+  "description": "Wrapper for PhysicsJS, which make basic game development as simple as possible.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
It'd be cool if we could set up basic unit testing. It looks like mocha is pretty good: https://mochajs.org/

Ideally, I'd say make sure we can import everything as very first unit test